src/ConsoleConnector/Commands/AddInstanceParamCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Autodesk.DataExchange.ConsoleApp.Commands.Options;
+using Autodesk.DataExchange.ConsoleApp.Helper;
+using Autodesk.DataExchange.ConsoleApp.Interfaces;
+using Autodesk.DataExchange.DataModels;
+
+namespace Autodesk.DataExchange.ConsoleApp.Commands
+{
+    internal class AddInstanceParamCommand : AddParamCommand
+    { 
+        public AddInstanceParamCommand(IConsoleAppHelper consoleAppHelper) : base(consoleAppHelper)
+        {
+            this.IsInstanceParameter = true;
+            this.Name = "AddInstanceParam";
```

Comment:
Please update this.Name to AddInstanceParameter, so that is the command typed by the user.